packages/pigeon/test/cpp_generator_test.dart

```diff
@@ -57,6 +57,7 @@ void main() {
       expect(code, contains('class Input'));
       expect(code, contains('class Output'));
       expect(code, contains('class Api'));
+      expect(code, contains('virtual ~Api() {}'));
```

Comment:
If this is the stray semi-colon being removed, this test doesn't cover that properly.

Reply:
Oops, good point